package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kintone-query-builder",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
I edited package.json manually without running `npm version` command, so package-lock.json was specifying 0.1.0.
`npm audit fix` fixes this inconsistency.